automation/scripts/publish_dev_spec_from_pr.sh

```
@@ -0,0 +1,38 @@
+  gh pr view "$PR_NUM_TRIM" --json title,body,number,headRefName,baseRefName,author,files,commits,additions,deletions,changedFiles,labels,assignees,reviewRequests,linkedIssues >> prs.json
+  echo -e "\n\n# Diff for PR $PR_NUM_TRIM\n" >> prs.diff
+  gh pr diff "$PR_NUM_TRIM" >> prs.diff
+done
+
+# Generate dev spec markdown
+bash automation/scripts/dev_spec_from_pr.sh prs.json prs.diff
+
+# Create dev-spec branch, commit, and push
+BRANCH=dev-spec/pr-$(echo "$PR_NUMBERS_RAW" | tr ',' '-')
+DEV_SPEC_FILE=DEV_SPEC_PR_$(echo "$PR_NUMBERS_RAW" | tr ',' '_').md
+git checkout -b $BRANCH
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)